src/mercury_engine_data_structures/formats/collision.py

```diff
@@ -38,14 +38,11 @@
 
 collision_formats = {
     "AABOX2D": Struct(
-        center=CVector2D,
-        unk=Float,
+        offset=CVector3D,
```

Comment:
call these `position` imo

Reply:
it's relative position, as opposed to global. in my experience relative position is the default when you just say "position"

Reply:
fair enough, fixed